specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,3118 @@
+        },
+        "error": {
+          "type": "object",
+          "description": "Document Error.",
+          "$ref": "#/definitions/TextAnalyticsError"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "type": "object",
+      "required": [
+        "error"
+      ],
+      "properties": {
+        "error": {
+          "type": "object",
+          "description": "Document Error.",
+          "$ref": "#/definitions/TextAnalyticsError"
+        }
+      }
+    },
+    "TextAnalyticsError": {
+      "type": "object",
+      "required": [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "InvalidRequest",
+            "InvalidArgument",
+            "InternalServerError",
+            "ServiceUnavailable",
+            "NotFound"
+          ],
+          "x-ms-enum": {
+            "name": "ErrorCodeValue",
+            "modelAsString": true
+          },
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Error message."
+        },
+        "target": {
+          "type": "string",
+          "description": "Error target."
+        },
+        "innererror": {
+          "$ref": "#/definitions/InnerError",
+          "description": "Inner error contains more specific information."
+        },
+        "details": {
+          "type": "array",
+          "description": "Details about specific errors that led to this reported error.",
+          "items": {
+            "$ref": "#/definitions/TextAnalyticsError"
+          }
+        }
+      }
+    },
+    "TextAnalyticsWarning": {
+      "type": "object",
+      "required": [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "LongWordsInDocument",
+            "DocumentTruncated"
+          ],
+          "x-ms-enum": {
+            "name": "WarningCodeValue",
+            "modelAsString": true
+          },
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Warning message."
+        },
+        "targetRef": {
+          "type": "string",
+          "description": "A JSON pointer reference indicating the target object."
+        }
+      }
+    },
+    "ExtractiveSummarizationTask": {
+      "properties": {
+        "parameters": {
+          "properties": {
+            "model-version": {
+              "default": "latest",
+              "type": "string"
+            },
+            "loggingOptOut": {
+              "default": true,
+              "type": "boolean"
+            },
+            "stringIndexType": {
+              "$ref": "#/definitions/StringIndexType"
+            },
+            "sentenceCount": {
+              "default": 3,
+              "type": "integer"
+            },
+            "sortBy": {
```

Comment:
looking at the REST API guidelines, the recommendation is to use `orderBy` instead of `sortBy`. @abhahn do you know if this could be renamed?

https://github.com/microsoft/api-guidelines/blob/azureRestUpdates/azure/Guidelines.md#query-options

Reply:
although this applies to collections, I wonder if it will be ok to unify,

Reply:
I think at this point it is too late to change for this preview version, but I can include this as a change to include in the next preview version to align with the guidelines.  I'm surprised this didn't come up during the API Review back in May.

Reply:
For sure. It could def happen in another version.
It could have been missed or because it is not a pageable sort? Idk...